File: src/main/java/com/shzhangji/timetable/model/Event.java
package com.shzhangji.timetable.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

import org.joda.time.DateTime;

import lombok.Data;

@Entity
@Data
public class Event {
	@Id
	@GeneratedValue
	private Integer id;
	private String title;
	private DateTime start;
	private DateTime end;
	@Column(updatable = false)
	private DateTime created;
	private DateTime updated;
}

File: timetable/models/event.py
from datetime import datetime

from sqlalchemy.orm import Mapped, mapped_column

from . import Base


class Event(Base):
    __tablename__ = 'event'

    id: Mapped[int] = mapped_column(primary_key=True)
    user_id: Mapped[int]
    title: Mapped[str]
    category_id: Mapped[int]
    start: Mapped[datetime]
    end: Mapped[datetime]
    created: Mapped[datetime]
    updated: Mapped[datetime]
